### PR TITLE
Bump Taipy version

### PR DESCRIPTION
**Pull request by gmarabout**
## Goal
Bump version of Taipy to 2.1.0

## Changes
* Pipfile now use explicit `taipy = ">=2.1.0"`
* Requires Python 3.11

## Not in this PR
* Better use of Taipy path in `package.json`
* Update `readme.md` about it

### CODE REVIEW

File: webcam/webui/package.json [hunk 1]
```diff
@@ -19,10 +19,10 @@
     "dependencies": {
         "@types/w3c-image-capture": "^1.0.7",
         "react": "^18.2.0",
-        "taipy-gui": "file:../../venv/lib/python3.10/site-packages/taipy/gui/webapp"
+        "taipy-gui": "file:/Users/gmarabout/.local/share/virtualenvs/demo-face-recognition-3GjZDeuO/lib/python3.11/site-packages/taipy/gui/webapp"
```
Comment (gmarabout) [inline webcam/webui/package.json:22]: **note to reviewers** We need a better way to update this (env variable? scripting?). This will come in another PR.

**Review by jrobinAV — COMMENTED** [review]
I believe the pipfile.lock should not be committed, right?

**Comment by gmarabout** [comment]
@jrobinAV The consensus is actually to commit it. See https://github.com/pypa/pipenv/issues/598 for instance.

**Comment by jrobinAV** [comment]
OK. We should commit it in our internal repos as well, then. 😄 

Thx !

**Pull request merged**